source/dotnet/DatabaseMigration/Scripts/202303081500_Create_Outbox_Table.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [OutboxMessage]
+(
+    [Id]       [uniqueidentifier]   NOT NULL,
+    [MessageType]     [nvarchar](max)      NOT NULL,
+    [Data]     [varbinary](max)            NOT NULL,
```

Comment:
Is varbinary human readable?
```suggestion
    [Data]     [nvarchar](max)            NOT NULL,
```

Reply:
No. The C# convertion is typically byte[] -> varbinary. Is it because you would like to take out the value (byte[]) and convert i needed?